node/core/av-store/src/lib.rs

```diff
@@ -477,56 +474,69 @@ fn get_next_chunk_pruning_time(db: &Arc<dyn KeyValueDB>) -> Option<NextChunkPrun
 }
 
 async fn run<Context>(mut subsystem: AvailabilityStoreSubsystem, mut ctx: Context)
```

Comment:
this function can be extracted (as a generic) into utils module

Reply:
let's wait until we're aware of the exact patterns we see